src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqModellingGroupRepository.kt

```diff
@@ -45,6 +45,16 @@ class JooqModellingGroupRepository(
                 .map { mapModellingGroup(it) }
     }
 
+    override fun getModellingGroups(ids: Array<String>): Iterable<ModellingGroup>
+    {
```

Comment:
Is this the right logic for what we're trying to do in getting modelling groups for a user - this is only returning 'current groups' - I assume that a member of an old group isn't automatically considered to be a member of its replacement.

Reply:
Good question. I think the reason for this `replaced_by` thing is in case the PI of a group changes, in which case we probably would want all members to get added to the new group. But it's easy to do that manually (in any case we have no mechanism for changing a group name right now, so it would be done by direct sql query, meaning members could also be updated at the same time.)

So yeah I wouldn't auto-forward memberships, but I also don't see any reason not to return all groups, i.e. drop the `and(MODELLING_GROUP.REPLACED_BY.isNull)` clause.

Reply:
So if we returned all groups, that would mean in this case that a replaced group which a user belonged to would be part of the list of groups which the user of the contrib portal can see. I think this would change the current behaviour, since it's currently initially getting the group returned by getModellingGroups, which does exclude replaced groups.

Reply:
OK, leaving this as is to avoid changing behaviour, but we should look into whether this is actually used